### PR TITLE
Fix duplicate declaration errors in anyPass example in REPL

### PR DESCRIPTION
**Pull request by aonghusonia**

### CODE REVIEW

File: src/anyPass.js [hunk 1]
```diff
@@ -23,11 +23,13 @@ var reduce = require('./reduce');
  * @see R.allPass
  * @example
  *
- *      var gte = R.anyPass([R.gt, R.equals]);
+ *      var isClub = R.propEq('suit', '♣');
+ *      var isSpade = R.propEq('suit', '♠');
+ *      var isblackCard = R.anyPass([isClub, isSpade]);
```
Comment (davidchambers) [inline src/anyPass.js]: Let's name this `isBlackCard` (with a capital "B").

**Comment by davidchambers** [comment]
LGTM. Either you should squash the commits or we'll let @buzzdecafe squash and merge.

**Comment by buzzdecafe** [comment]
:dromedary_camel:

**Comment by CrossEye** [comment]
:herb:

**Pull request merged**